src/Foundatio/Storage/FileStorageOptionsBase.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Foundatio.Serializer;
+using Microsoft.Extensions.Logging;
+
+namespace Foundatio.Storage {
+    public abstract class FileStorageOptionsBase {
```

Comment:
Can we also quickly make this change to in memory storage as well (even if we don't add any logging for now,